src/main/resources/META-INF/mods.toml

```diff
@@ -0,0 +1,52 @@
+# This is an example mods.toml file. It contains the data relating to the loading mods.
```

Comment:
You probably don't need all the example text, especially stuff that's very easily outdated like the comments on what the current versions are.

The short description of each field is fine, though.